doc/use_apps/verify_quote.rst

```diff
@@ -1,7 +1,7 @@
 Verifying Quote
 ===============
 
-A client can verify the validity of the SGX quote of the CCF node that it connects to. This can be done using the ``verify_quote.sh`` script installed as part of the CCF install or ccf Python package (see :doc:`/use_apps/python_tutorial`).
+A client can verify the validity of the SGX quote of the CCF node that it connects to. This can be done using the ``verify_quote.sh`` script installed as part of the CCF install or ``ccf`` Python package.
```

Comment:
Is there any documentation for the verify quote functionality in the python library? https://microsoft.github.io/CCF/main/audit/python_library.html seems to only cover ledger operations

Reply:
The `verify_quote.sh` utility is a simple bash script that we bundle with our `ccf` Python package as a convenient way to ship the utility to users. We haven't got a Python version for it but will add one if users ask for it (no demand so far AFAIK).